docker-compose.windows.yml

```diff
@@ -0,0 +1,47 @@
+version: '3'
+services:
+  start-test-agent.windows:
+    build:
+      context: ./tracer/build/_build/docker/
+      dockerfile: wait-for-dependencies-windows.dockerfile
+    image: andrewlock/wait-for-dependencies-windows
+    depends_on:
+    - test-agent.windows
+    environment:
+    - TIMEOUT_LENGTH=120
+    command: test-agent.windows:8126
+
+  test-agent.windows:
+    build:
+      context: ./tracer/build/_build/docker/
+      dockerfile: test-agent.windows.dockerfile
+    image: dd-trace-dotnet/ddapm-test-agent-windows
+    volumes:
+    - ./tracer/build/smoke_test_snapshots:c:/snapshots
+    - ./tracer/build_data/snapshots:c:/debug_snapshots
+    ports:
+    - "8126:8126"
+    environment:
+    - SNAPSHOT_CI=1
+    - SNAPSHOT_IGNORED_ATTRS=span_id,trace_id,parent_id,duration,start,metrics.system.pid,meta.runtime-id,metrics.process_id
```

Comment:
Maybe for a followup PR, but I wonder if you can remove this line now?

Reply:
Not yet, it's not in the latest release yet